### ISSUE TITLE
Discord

### ISSUE THREAD

**Issue by ahcenezdh** [esx-overextended/es_extended#30]
Hey, would be cool to have a discord !

**Comment by mahanmoulaei** [comment]
Of course, a Discord server would be nice, but until I publish the first stable version with all of the enhancements I need to be applied, I won't be setting up a related Discord to esx-overextended. However, I do have a discord for my other organization that anyone can join there **_temporarily_** until I set up a related server...
https://discord.gg/5JzWm4ndsB
